src/index.html

```diff
@@ -9,10 +9,56 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="styles/style.css"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
     />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="body">
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__image"></div>
+      <div class="card__name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+
+      <div class="card__code">Product code: 195434</div>
+      <div class="review">
+        <div class="stars">
+          <div class="stars__star-active"></div>
+          <div class="stars__star-active"></div>
+          <div class="stars__star-active"></div>
+          <div class="stars__star-active"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="review__reviews">Reviews: 5</div>
+      </div>
+      <div class="card__price">
+        <div class="card__price-key">Price:</div>
+        <div class="card__price-value">$2,199</div>
+      </div>
+
+      <a
+        type="submit"
```

Comment:
The `type` attribute is not valid for the `<a>` element. If you want to style this link as a button, consider using a `<button>` element or remove the `type` attribute.